lib/exception_notifier/email_notifier.rb

```diff
@@ -55,6 +72,13 @@ def background_exception_notification(exception, options={}, default_options={})
 
             compose_email
           end
+  
+          helper_method :shrink_contents
+           def shrink_contents(sections_content)
```

Comment:
This method should also have `else` condition because in the existing code if condition is not true then it is returning `nil` which is undesired.